cmd/scheduler/main.go

```diff
@@ -61,7 +61,7 @@ func init() {
 	rootCmd.Flags().Int32Var(&config.DefaultResourceNum, "default-gpu", 1, "default gpu to allocate")
 	rootCmd.Flags().StringVar(&config.NodeSchedulerPolicy, "node-scheduler-policy", util.NodeSchedulerPolicyBinpack.String(), "node scheduler policy")
 	rootCmd.Flags().StringVar(&config.GPUSchedulerPolicy, "gpu-scheduler-policy", util.GPUSchedulerPolicySpread.String(), "GPU scheduler policy")
-	rootCmd.Flags().StringVar(&config.MetricsBindAddress, "metrics-bind-address", ":9395", "The TCP address that the scheduler should bind to for serving prometheus metrics(e.g. 127.0.0.1:9395, :9395)")
```

Comment:
This change will cause problems. Previously, MetricsBindAddress allowed users to pass the entire address (address+port) as a parameter, not just the port.

For example, users do not want MetricsBindAddress to be exposed as "0.0.0.0", which will bring other risks, security, etc.